src/clientsettingsdlg.cpp

```diff
@@ -1063,25 +1063,31 @@ void CClientSettingsDlg::OnSndCrdBufferDelayButtonGroupClicked ( QAbstractButton
     UpdateDisplay();
 }
 
+/// @method
+/// @brief Sets upstream rate label to current upload rate if the client is connected, else resets label
 void CClientSettingsDlg::UpdateUploadRate()
 {
-    // update upstream rate information label
-    lblUpstreamValue->setText ( QString().setNum ( pClient->GetUploadRateKbps() ) );
-    lblUpstreamUnit->setText ( "kbps" );
+    // update upstream rate information label if needed
+    if ( pClient->IsConnected() )
```

Comment:
This is
```
bool IsConnected() { return Channel.IsConnected(); }
```

Reply:
Yes. That's from https://github.com/jamulussoftware/jamulus/pull/2550/files#diff-d5cde46c72d0c085d1d2c0ca59957407b17931c2f194a33268b3535169006629R1042

A potential semantic change. Probably worth reverting to IsRunning() unless proven to be incorrect.

Reply:
I'd like to hear a second OK/not OK from someone nevertheless.